run_kraken.py

```diff
@@ -279,9 +327,22 @@ def main(cfg):
                         # Inject time skew chaos scenarios specified in the config
                         elif scenario_type == "time_scenarios":
                             time_scenarios(scenarios_list, config)
+                        elif scenario_type == "litmus_scenarios":
+                            if not litmus_installed:
+                                common_litmus.install_litmus(litmus_version)
+                                common_litmus.perform_experiment(litmus_version)
+                                litmus_installed = True
+                            litmus_namespaces = litmus_scenarios(scenarios_list, config,
+                                                                 litmus_namespaces)
 
             iteration += 1
             logging.info("")
+        if litmus_installed:
+            for namespace in litmus_namespaces:
+                common_litmus.delete_chaos(namespace)
+            common_litmus.delete_experiments()
```

Comment:
We might want to make this optional with the default set to False, it's useful to keep them around for debugging. Thoughts?

Reply:
Yeh I think that is a good idea, adding an extra flag in the config to delete or not based on the run status